exercises/practice/complex-numbers/.meta/src/reference/java/ComplexNumber.java

```diff
@@ -46,12 +48,4 @@ ComplexNumber conjugate() {
     ComplexNumber exponentialOf() {
         return new ComplexNumber(Math.cos(imaginary), Math.sin(imaginary)).multiply(Math.exp(real));
     }
-
-    private ComplexNumber divide(double factor) {
-        return new ComplexNumber(real / factor, imaginary / factor);
-    }
-
-    private ComplexNumber multiply(double factor) {
-        return new ComplexNumber(factor * real, factor * imaginary);
```

Comment:
There is a compile error because this method is still being used on line 49 (see the [Lint output](https://github.com/exercism/java/actions/runs/11641679071/job/32422104274?pr=2851#step:4:673)).